llvm/lib/Support/Timer.cpp

```diff
@@ -305,14 +306,24 @@ TimerGroup::~TimerGroup() {
     PrintQueuedTimers(*OutStream);
   }
 
+  auto unlink = [&]() {
+    *Prev = Next;
+    if (Next)
+      Next->Prev = Prev;
+  };
+
+  // If the managed instance is already dead, it means we're in the CRT
```

Comment:
I am confused by this comment.

`llvm::TimerGroup::acquireTimerGlobals()` frees ManagedStatic. Is the leak due to timerLock() re-constructing the ManagedStatic?

Reply:
Yes, exactly.

Reply:
I will clarify the comment.